src/countBullsAndCows.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const countBullsAndCows = (randomNumber, playerNumber) => {
+  let countBulls = 0;
+  let countCows = 0;
+
+  playerNumber
+    .toString()
+    .split('')
+    .forEach((num, index) => {
+      if (randomNumber.includes(num)) {
+        if (randomNumber.indexOf(num) === index) {
+          countBulls++;
+        } else {
+          countCows++;
+        }
```

Comment:
I guess it's better to avoid nested if-conditions here

Reply:
Yep agree, I will try to rethink this part.